ports/esp32/modules/_boot.py

```diff
@@ -5,6 +5,7 @@
 try:
     if bdev:
         uos.mount(bdev, '/')
+	print("Mounted /")
```

Comment:
indentation looks wrong. tab? one space missing?

Reply:
oh.. yes. thx Will fix it.